container-chains/templates/simple/node/src/command.rs

```diff
@@ -333,8 +319,7 @@ pub fn run() -> Result<()> {
 				let parachain_account =
 					AccountIdConversion::<polkadot_primitives::AccountId>::into_account_truncating(&id);
 
-				let state_version = Cli::native_runtime_version(&config.chain_spec).state_version();
-				let block: Block = generate_genesis_block(&*config.chain_spec, state_version)
+				let block: Block = generate_genesis_block(&*config.chain_spec, sp_runtime::StateVersion::V1)
```

Comment:
Change this to reading it from runtime!

Reply:
To be fair this is only used for logging purposes. I can try to change it to something where we read it from the runtime, but it would take a bit more to spin up the node. Thoughts? @tmpolaczyk @nanocryk

Reply:
I see, this is used to print the genesis block hash. It sounds useful, why do you say it will take more time to spin up a node?

Reply:
we experienced that when we changed the export-genesis-state from using a static state version value to reading it from the runtime. I dont think this should be the case if it is only done for logging purposes.

I think the best way is to print both state v0 and state v1 genesis states, and let the user read whichever is useful for it.